index.html

```diff
@@ -1430,10 +1430,16 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
       Promise&lt;void&gt; push(NFCMessageSource message, optional NFCPushOptions options);
     };
 
-    [Constructor(), SecureContext, Exposed=Window]
+    [Constructor(optional NFCReaderOptions options), SecureContext, Exposed=Window]
     interface NFCReader : EventTarget {
-      Promise&lt;void&gt; watch(optional NFCWatchOptions options);
+      readonly attribute USVString url;
+      readonly attribute NFCRecordType recordType;
+      readonly attribute USVString mediaType;
+      readonly attribute NFCAccept accept;
+
       attribute EventHandler onreading;
+
+      Promise&lt;void&gt; watch();
```

Comment:
IMHO removing the argument of `watch()` and passing it to the constructor is half-breed solution. 
I would use either `watch()` with the options, or constructor with the function passed together with options.

Reply:
This is an incremental change to make sure it all works, `watch()` will be removed soon enough and it will be activated when event listener is attached

Reply:
Incremental changes SGTM. I like the plan of activating when the event listener is attached.

@zolkis You might want to avoid using analogies like "half-breed", which could be considered demeaning.